version/version.go

```diff
@@ -16,5 +16,18 @@ limitations under the License.
 package version
 
 var (
-	Version = "v0.0.0"
+	// This variable shows the user any additional changes to files made that aren't on the branch.
+	GitDirty string
```

Comment:
this should probably be `bool`, wdyt?

Reply:
Interesting - I thought you wanted the output of `git diff --stat` be appended to the string, which is why it was a string. Makes sense - when there is content within the string, the boolean should be set to true, otherwise false. I will go ahead an implement. Thanks for reviewing my code, @eguzki !

Reply:
`GitDirty bool` is just a flag to know whether `-dirty` suffix needs to be added to the commitID or not

Reply:
Sure, it turns out it can be processed directly in Makefile, so I have actually removed it and simplified init() .